src/coreclr/jit/codegenriscv64.cpp

```diff
@@ -4901,39 +4899,40 @@ void CodeGen::genPutArgStk(GenTreePutArgStk* treeNode)
         argOffsetMax = compiler->lvaOutgoingArgSpaceSize;
     }
 
-    bool isStruct = (targetType == TYP_STRUCT) || (source->OperGet() == GT_FIELD_LIST);
```

Comment:
Why not `GT_FIELD_LIST`?

Reply:
As I understand, all GT_FIELD_LIST nodes have TYP_STRUCT type.
@SingleAccretion If I misunderstand, please let me know.
Thank you.

Reply:
> As I understand, all GT_FIELD_LIST nodes have TYP_STRUCT type.

That is correct.